web-ui/controller/dune

```
@@ -10,6 +10,7 @@
   ppx_deriving_yojson.runtime
   prometheus-app
   capnp-rpc-unix
-  ocaml-ci-web.view)
+  ocaml-ci-web.view
+  client-utilities)
```

```suggestion
  ocaml-ci-client-utilities)
```
Same: can be ignored.